baremetal/metal3machine_manager_test.go

```diff
@@ -106,7 +106,7 @@ func consumerRef() *corev1.ObjectReference {
 	return &corev1.ObjectReference{
 		Name:       metal3machineName,
 		Namespace:  namespaceName,
-		Kind:       "M3Machine",
+		Kind:       "Metal3Machine",
```

Comment:
"Metal3Machine" string is used so many times in this file, would be nice have it in const at top of the file.

Reply:
yes, I made unified variable for metal3Machine Kind. Please check